types/v56/types.go

```diff
@@ -2317,25 +2328,55 @@ type VimObjectRefs struct {
 	VimObjectRef []*VimObjectRef `xml:"VimObjectRef"`
 }
 
+// Type: VimObjectRefsType
+// Namespace: http://www.vmware.com/vcloud/extension/v1.5
+// https://vdc-repo.vmware.com/vmwb-repository/dcr-public/7a028e78-bd37-4a6a-8298-9c26c7eeb9aa/09142237-dd46-4dee-8326-e07212fb63a8/doc/doc/types/VimObjectRefsType.html
+// Description: List of VimObjectRef elements.
+// Since: 0.9
+type VimObjectRefsCreate struct {
+	VimObjectRef []*VimObjectRefCreate `xml:"vmext:VimObjectRef"`
+}
+
+// Type: VMWExternalNetworkType
+// Namespace: http://www.vmware.com/vcloud/extension/v1.5
+// https://vdc-repo.vmware.com/vmwb-repository/dcr-public/7a028e78-bd37-4a6a-8298-9c26c7eeb9aa/09142237-dd46-4dee-8326-e07212fb63a8/doc/doc/types/VMWExternalNetworkType.html
+// Description: External network type.
+// Since: 1.0
+type ExternalNetworkCreate struct {
```

Comment:
Are you sure the same structure (old or new) cannot be used in both places?
This makes it  complicated.

Reply:
Please find solution better, I already spent a day figure out one working solution. In general go don't work with namespaces - request hangs up for 2 years minimum